src/System.Runtime/tests/System/Reflection/PointerTests.cs

```diff
@@ -86,6 +86,7 @@ public void PointerValueRoundtrips(int value)
 
         [Theory]
         [MemberData(nameof(Pointers))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Mono)]
```

Comment:
What does Mono do for these tests? Shouldn't there be a bug tracking this instead? This test is testing how to reflection-set a pointer-typed field. There's no other way to that.

Reply:
This is a Mono bug as far as I can tell. We should keep it in Mono's .rsp files and track it there.

Reply:
Yeah, leaving it in the rsp, thanks